cloudmarker/test/test_esstore.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for EsStore plugin."""
+
+import unittest
+from unittest import mock
+
+from elasticsearch import ElasticsearchException
+
+from cloudmarker.stores import esstore
+
+
+class EsStoreTest(unittest.TestCase):
+    """Tests for EsStore plugin."""
+
+    @mock.patch('cloudmarker.stores.esstore.Elasticsearch.bulk')
+    def test_happy_flow(self, mock_bulk):
```

Comment:
I think we can make the `@mock.patch()` simpler.

- Update `esstore.py` to `import elasticsearch`.
- Then in these test methods, `@mock.patch('esstore.Elasticsearch.bulk')`.

I guess this is one more reason to follow the convention of importing modules (not classes and functions) in the application code. It makes it easier to patch them with a mock in the test methods.

Reply:
@r0hi7 feel free to update import paths in `esstore.py` so that the patches in this file could be made simpler.

Reply:
Can I do it in a separate commit? As this would also require to change the esstore.py file.

Reply:
I would have preferred the `esstore.py` change in this commit itself. The `esstore.py` change should be straightforward. However, if you prefer to do it in a separate commit, that's fine too.